src/components/dev-hub/media-block.js

```diff
@@ -43,7 +43,7 @@ const MediaBlockContainer = styled('div')`
 const MediaWrapper = styled('div')`
     grid-area: image;
     @media ${screenSize.largeAndUp} {
-        margin-right: ${size.medium};
+        ${({ reverse }) => !reverse && `margin-right: ${size.medium};`};
```

Comment:
Do we need to handle left margin instead for reverse?

Reply:
Good point!